src/game/default/g_cmd.c

```diff
@@ -729,6 +754,10 @@ static void G_Vote_f(g_entity_t *ent) {
 
 	ent->client->locals.persistent.vote = VOTE_YES; // client has implicitly voted
 	g_level.votes[VOTE_YES] = 1;
+	
+	if (ent->client->locals.persistent.admin) {
+		g_level.votes[VOTE_YES] = 100;	// admin votes always succeed...
```

Comment:
That makes sense. They can call a vote, but since the caller automatically votes yes the vote is called and then succeeded. I guess we could just not have the caller vote yes automatically if they're an admin...